src/pages/Login.vue

```diff
@@ -132,17 +128,20 @@ export default {
           throw new Error(this.$t('somethingWentWrong'))
         }
 
-        await callApi('/users/me', res.token).then((resp) => {
-          this.auth = {
-            token: res.token,
-            firstname: resp.user.firstName,
-            lastname: resp.user.lastName,
-            email: resp.user.email,
-            id: resp.user._id,
-            authenticated: true
-          }
-          this.$router.go(-1)
-        })
+        await callApi('/users/me', res.token)
+          .then((resp) => {
+            this.auth = {
+              token: res.token,
+              firstname: resp.user.firstName,
+              lastname: resp.user.lastName,
+              email: resp.user.email,
+              id: resp.user._id,
+              authenticated: true
+            }
+          })
+          .finally(() => {
+            this.$router.go('/')
```

Comment:
Why do you send the user to the index page now ?

Reply:
Ahh I see this is not working anyway. The Question is where to go after login.

Reply:
Ideally, where the user wanted to go, and that should indeed be '-1' like it was before

Reply:
Okay, changed it back as it was

Reply:
Did you try it ? I think right now it doesn't work. Currently the previous route is some random page the user visited, before deciding to go to the login screen. Let's change it to '/profile'

Reply:
I used /profile. In my opinion, this makes the most ideal way.